tests/unit/sys/test_output.cxx

```diff
@@ -294,3 +295,21 @@ TEST_F(OutputTest, DummyJustPrint) {
   EXPECT_EQ("", test_buffer.str());
   EXPECT_EQ("", buffer.str());
 }
+
+TEST_F(OutputTest, FormatInd3Ddefault) {
+  Ind3D ind(11, 2, 3);
+
+  Output local_output;
+  local_output.write("{}", ind);
+
+  EXPECT_EQ(buffer.str(), "(1, 1, 2)");
+}
+
+TEST_F(OutputTest, FormatInd3Di) {
+  Ind3D ind(11, 2, 3);
+
+  Output local_output;
+  local_output.write("{:i}", ind);
```

Comment:
Worth adding `:c` version as well?

How about testing an invalid format behaves as expected as well?

Reply:
Added tests for `{:c}` and the invalid `{:b}`